src/pages/Login/index.js

```diff
@@ -222,6 +288,132 @@ function Login(props) {
                         : classes.googleLogin
                     }
                   />
+                  <Typography
+                    variant="subtitle1"
+                    fontSize="12px"
+                    textAlign="center"
+                  >
+                    <div
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "space-between",
+                        gap: "10px",
+                        padding: "20px 0",
+                      }}
+                    >
+                      <div
+                        style={{
+                          height: "1px",
+                          width: "40%",
+                          backgroundColor: "gray",
+                        }}
+                      ></div>
+                      <Typography
+                        variant="body1"
+                        style={{
+                          fontSize: "12px",
+                          color: "black",
+                          whiteSpace: "nowrap",
+                          backgroundColor: "white",
+                        }}
+                      >
+                        or login with Username and Password
+                      </Typography>
+                      <div
+                        style={{
+                          height: "1px",
+                          width: "40%",
+                          backgroundColor: "gray",
+                        }}
+                      ></div>
+                    </div>
+                  </Typography>
+                  <form onSubmit={handleSubmit}>
+                    <Box
+                      sx={{
+                        marginBottom: errors.username ? "40px" : "25px",
+                        marginBlock: "10px",
+                      }}
+                    >
+                      <TextField
+                        fullWidth
+                        id="username"
+                        name="username"
+                        placeholder="Username"
+                        label={user.error ? "Username" : ""}
+                        value={formData.username}
+                        onChange={handleChange}
+                        error={!!errors.username}
```

Comment:
This won't be `error={!!errors.username}`, it would be `error={errors.username}`